doc/source/methods/Anchors.ipynb

```diff
@@ -64,7 +64,9 @@
     "\n",
```

Comment:
<p>computational expensive --&gt; computationally expensive</p><p>replace by the --&gt; replaced by the</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/432/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>